modules/end-user-guide/partials/proc_building-an-image-for-an-ide-based-on-the-intellij-platform.adoc

```diff
@@ -0,0 +1,160 @@
+[id="building-an-image-for-an-ide-based-on-the-intellij-platform_{context}"]
+= Building an image for an IDE based on the IntelliJ Platform
+
+This procedure describes how to build an image for an IDE based on the IntelliJ Platform up to version 2020.3. See the link:https://github.com/che-incubator/jetbrains-editor-images/blob/main/doc/Compatible-IDE.md[list of compatible IDEs].
+
+.Prerequisites
+
+* The build host has at least 2 GB of available RAM.
+
+* The build host has an installation of following tools:
```

Comment:
`The build host has an installation of following tools:` = `The build host makes an installation of following tools:`

Reply:
```suggestion
* The following tools are installed on the build host:
```

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#general-formatting
_"When writing prerequisites, be as clear and concise as possible. You can use the passive voice, if necessary, to achieve that end."_
BTW, "Git is available" is also passive voice.

Reply:
Oh that's good to know.